src/udfs/ndarray_udfs/array_count.py

```diff
@@ -0,0 +1,74 @@
+# coding=utf-8
+# Copyright 2018-2020 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pandas as pd
+import numpy as np
+from src.udfs.ndarray_udfs.abstract_ndarray_udfs import AbstractNdarrayUDF
+
+
+class Array_Count(AbstractNdarrayUDF):
+
+    @property
+    def name(self) -> str:
+        return 'Array_Count'
+
+    def exec(self, inp: pd.DataFrame):
+        # input should just be a single column
+        # (Data Series or DF with one column)
+
+        # pre conditions: #####################################################
+        # check if numpy array, else throw
+        # check number of dimensions of numpy array
+        # check number of dimensions of searchElement if array
+        # if number of dimensions of searchElement not less by 1 than numpy
+        # array throw
+
+        # loop over and apply count function
+        # convert to pd data frame and then add index as a column (optional)
+
+        # inp (1 column), label: ["A"]
+        # row 0: [["A"], ["A"], ["B"]]
+        # row 1: [["A"], ["B"], ["B"]]
+        # return: [2,
```

Comment:
Does `array_count` return one number (sum) or an array of count in each dimension?

Reply:
I'll fix the comments for better understanding. It will return a sum for each tuple. The idea is to flatten along the first dimension and then count the search element in this flattened array. 
For example, a tuple of shape (3,4,5) will be flattened into three (4,5) elements. And the search key is assumed to be of shape (4,5), else we throw an error. and this is done for all the tuples in the batch.

Reply:
So the return result will be just one integer.

Reply:
Yes, a single integer for each tuple.